controllers/app.controllers.js

```diff
@@ -78,3 +80,18 @@ exports.updateArticleById = (request, response, next) => {
       next(err);
     });
 };
+
+exports.deleteCommentById = (request, response, next) => {
+  const { comment_id } = request.params;
+  const validCommentIdQuery = validateCommentId(comment_id);
+  const deleteCommentQuery = deleteCommentFromDB(comment_id);
```

Comment:
this is overcomplicated. We dont need to check its a valid comment id - if we using RETURNING * on our delete query we can see if there was a comment deleted